design/design-doc-observe-only-resources.md

```diff
@@ -2,7 +2,7 @@
 
 * Owners: Hasan Turken (@turkenh)
 * Reviewers: Crossplane Maintainers
-* Status: Draft
+* Status: Defunct (superseded by [ignore changes])
```

Comment:
We move a defunct design under https://github.com/crossplane/crossplane/tree/master/design/defunct